hardhat.config.ts

```diff
@@ -104,7 +104,7 @@ const config: HardhatUserConfig = {
       forking: {
         enabled: true,
         url: "https://xdai-archive.blockscout.com/",
-        blockNumber: 17587738,
+        blockNumber: 17568384,
```

Comment:
@jacobrosenthal I notice that my number is lower than what you set. Was your number supposed to be contract creation or when your account initialized?

Reply:
Alright, I'll undo this then.
```suggestion
        blockNumber: 17587738,
```